src/exchange/coinbase-processor/commands/start.go

```diff
@@ -32,6 +32,10 @@ func startCommand() *cobra.Command {
 				sugarLogger.Fatal(err)
 			}
 
+			if config.Coinbase.UseSandbox {
```

Comment:
again, why hardcoded

Reply:
what do you mean hardcoded? @sepisoad

Reply:
the values are set in the config file and can be overridden with Environment varaiuble